options/rtdl/generic/linker.hpp

```diff
@@ -167,6 +167,11 @@ struct SharedObject {
 	bool wasInitialized;
 
 	Scope *objectScope;
+
+	// PHDR related stuff
+	void *phdr_pointer = nullptr;
+	size_t phdr_entry_size = 0;
+	size_t phdr_count = 0;
```

Comment:
Please use camelCase for consistency with the other members.

Reply:
Add a comment that we currently only set these for the main executable. After the PR is merged, add an issue to add them in the general case.